fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

```diff
@@ -120,9 +134,16 @@ public Column(String name, Type type, boolean isKey, AggregateType aggregateType
         this.isAggregationTypeImplicit = false;
         this.isKey = isKey;
         this.isAllowNull = isAllowNull;
-        this.defaultValue = defaultValue;
+        if (defaultValueDef != null) {
+            if (defaultValueDef.expr instanceof StringLiteral) {
+                this.defaultValue = ((StringLiteral) defaultValueDef.expr).getValue();
+            } else if (defaultValueDef.expr instanceof NullLiteral) {
+                this.defaultExpr = null;
```

Comment:
This is inconsistent with the following function.
Return None when defaultExpr == null in `getDefaultValueType()`.